src/gbd_mapping/coverage_gap.py

```diff
@@ -12,8 +12,8 @@
 
 
 coverage_gaps = CoverageGaps(
-    lack_of_breastfeeding_promotion=CoverageGap(
-        name='lack_of_breastfeeding_promotion',
+    lack_of_vitamin_a_fortification=CoverageGap(
```

Comment:
I put a sort argument in the coverage gaps some time ago to keep the order stable.  Will you make sure it's still there?

Reply:
it was there but the generator was bypassing it and using the unsorted list. Just put it back.